.github/renovate.json

```
@@ -10,5 +10,24 @@
     ":prNotPending",
     ":rebaseStalePrs",
     ":automergeDigest"
+  ],
+  "customDatasources": {
+    "openstackhelm": {
+      "defaultRegistryUrlTemplate": "https://tarballs.opendev.org/openstack/openstack-helm/",
+      "format": "html",
+      "transformTemplates": [
+        "{\"releases\": $map($filter($.releases, function($v) {$match($v.version, /^{{packageName}}-/)}), function($v) { {\"version\": $match($v.version, /^{{packageName}}-([0-9.]+).tar.gz$/).groups[0]} } ) }"
```

```suggestion
        "{\"releases\": $map($filter($.releases, function($v) {$match($v.version, /^{{packageName}}-/)}), function($v) { {\"version\": $match($v.version, /^{{packageName}}-([0-9.]+)\\.tar\\.gz$/).groups[0]} } ) }"
 ```

We may want to escape the literal dots here, but I bet the original version still works so nbd

---

Yeah I dunno. I used the recommended website to craft the rule and then escaped it until the checker was happy. Not sure if it'll do the right thing yet.

---

What should we do with the PR?